java/BankingApplication.java

```diff
@@ -0,0 +1,89 @@
+package calculator;
+
+import java.text.NumberFormat;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import calculator.Loan.LoanType;
+
+public class BankingApplication {
+
+	enum UserChoice {
+		ADMIN, CUSTOMER, STOP
+	}
+
+	public static void main(String args[]) {
+
+		Loan loan = new Loan();
+		NumberFormat nf = NumberFormat.getInstance();
+
+		String choice = "";
+		while (!choice.toUpperCase().equals(UserChoice.STOP)) {
+			try {
+				Scanner scan = new Scanner(System.in);
+				System.out.println("Admin " + " Customer " + " Stop");
+				choice = scan.next();
+				UserChoice userOrAdmin = UserChoice.valueOf(choice.toUpperCase());
+
+				if (userOrAdmin == UserChoice.ADMIN) {
+					System.out.println("Enter the Type of Loan");
+					System.out.println(" Home " + " Personal " + " Car");
+					LoanType loantype = LoanType.valueOf((scan.next()).toUpperCase());
+
+					System.out.println("ViewInterest " + "ChangeInterest");
```

Comment:
Whenever displaying any values to user, be user friendly. User may or may not understand Java naming conventions.
Also instead of using '+' for concatenation you can also use other String methods such as String formatter.